src/components/CoordinatorHome.js

```diff
@@ -56,7 +56,7 @@ const CoordinatorHome = observer(({ assembly }) => (
 
       <tbody>
         {assembly.coordinator_registration.information.participants.map(participant =>
-          <tr key={participant.uuid} >
+          <tr key={participant.uuid} onClick={() => assembly.participant_history.watch(participant.uuid)} >
```

Comment:
This syntax is confusing. When you click the watch method within CoordinatorParticipantHistory is going to hear the uuid and then render a component?